roles/osm-vector-maps/tasks/install.yml

```diff
@@ -181,8 +181,17 @@
       - iiab-make-init.py
       - iiab-update-map
 
-- name: Install the World Map to zoom 10
-  shell: iiab-install-map-region
+- name: Link to the World Map to zoom 6
+  file:
+      src: '{{ vector_map_path }}/installer/planet_z0-z6_2019.mbtiles'
+      dest: '{{ vector_map_path }}/viewer/tiles/planet_z0-z6_2019.mbtiles'
+      state: link
+  when: osm_vector_maps_enabled | bool
+
+- name: Copy the map abbreviated satellite images
+  get_url:
+    url: "{{ map_catalog_url }}/satellite_z0-z6_v3.mbtiles"
+    dest: '{{ vector_map_path }}/viewer/tiles/satellite_z0-z6_v3.mbtiles'
```

Comment:
Can this be moved to line 92 for better grouping? Don't really need the when: osm_vector_maps_enabled in install.yml